src/bun.js/bindings/BunProcess.cpp

```diff
@@ -2859,6 +2860,30 @@ static Structure* constructCPUUsageStructure(JSC::VM& vm, JSC::JSGlobalObject* g
         offset);
     return structure;
 }
+
+static Structure* constructResourceUsageStructure(JSC::VM& vm, JSC::JSGlobalObject* globalObject)
+{
+    JSC::Structure* structure = globalObject->structureCache().emptyObjectStructureForPrototype(globalObject, globalObject->objectPrototype(), 16);
```

Comment:
I think this needs to be createStructure instead of setting IsPrototype since we use the structure directly without making it a prototype of another object.

Reply:
Looking into this more, our usage of `structureCache.emptyObjectStructureForPrototype` for result object structures in BunProcess.cpp is okay. In webkit the same is done for iterator result object structures `{ value, done }`.

https://github.com/oven-sh/WebKit/blob/b91aacf1fb87c5771e5cb4f97cd2ba602b7fba96/Source/JavaScriptCore/runtime/IteratorOperations.cpp#L136-L145